crates/ra_hir_ty/src/tests/regression.rs

```diff
@@ -533,3 +533,45 @@ fn foo(b: Bar) {
     "###
     );
 }
+
+#[test]
+fn issue_4053_diesel_where_clauses() {
+    std::env::set_var("CHALK_DEBUG", "1");
```

Comment:
Should this be removed? It won't be scoped to a single test.

Reply:
oh, yeah